### ISSUE TITLE
[Feature] Forcing WorldEdit Selection (If Plugin Exists) (Configurable)

### ISSUE THREAD

**Issue by clucky** [t00thpick1/Residence#18]
Oftentimes it creates confusion for the players on my server to have access to two completely separate selection plugins which are used for nearly the same thing. Such issues are generally pertained to players neglecting to remember to type `/res select worldedit`, resulting in them claiming the absolute wrong area. This in addition to there not currently being a refund feature means that either they loose out on their money/land or a staff member has to get involved and attempt to correct the issue. 

Being WorldEdit integration is already being used by Residence, I do not believe it would be too much further to force players to use WorldEdit instead of the default Residence selection.

Other than less confusion, some other pros to forcing WorldEdit selection is access to a wider variety of selection commands such as `//size` and `//contract`. In addition to this, forcing all players to use the same selection plugin means that things such as [WorldEdit Selection Visualizer](http://dev.bukkit.org/bukkit-plugins/worldedit-selection-visualizer/) will be used by all players. As well as WorldEdit Selection Visualizer, there is the rarely updated WorldEdit CUI mod, which I have a preference over compared to WorldEdit Selection Visualizer.

The configuration option would of course be boolean, more than likely named "ForceWE", "ForceWESelection" or "ForceWorldEditSelection".

**Comment by insert3coins** [comment]
pointless if a server only has residence and not worldedit.. just saying

**Comment by clucky** [comment]
Most servers have both Residence and WorldEdit, however, if your server does not have WorldEdit, the feature would obviously not be enabled. Even still, if your server did have WorldEdit and you wanted both Residence selection and WorldEdit selection  enabled, you would simply set the `ForceWorldEditSelection` configuration setting to `false`... Which I personally would prefer that the configuration setting for `ForceWorldEditSelection` be `false` by default anyway.

**Comment by insert3coins** [comment]
yea i find worldedit not useful for my server thus is why i don't need it, 100% survival babeh :)

**Comment by MarcusCheck** [comment]
Yeah, I have a prison world that I use WE on and a Free world that I use WE and Residence on.  This would be helpful option to have.
